pages/setup/finish.page.tsx

```diff
@@ -0,0 +1,112 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import React, { useEffect } from 'react';
+import { Button } from '@mui/material';
+import { Trans, useTranslation } from 'react-i18next';
+import { makeGetServerSideProps } from 'pages/api/utils/pagePropsHelpers';
+import { SetupPage } from 'src/components/Setup/SetupPage';
+import { LargeButton } from 'src/components/Setup/styledComponents';
+import useGetAppSettings from 'src/hooks/useGetAppSettings';
+import makeSsrClient from 'src/lib/apollo/ssrClient';
+import {
+  DefaultAccountListDocument,
+  DefaultAccountListQuery,
+  useUpdateSetupPositionMutation,
+} from './Finish.generated';
+
+interface PageProps {
+  defaultAccountListId: string;
+}
+
+// This is the last page of the tour, and it lets users choose to go to the
+// import page. It is always shown.
+const FinishPage: React.FC<PageProps> = ({ defaultAccountListId }) => {
+  const { t } = useTranslation();
+  const { appName } = useGetAppSettings();
+  const { push } = useRouter();
+  const [updateSetupPosition] = useUpdateSetupPositionMutation();
+
+  const setSetupPosition = (setupPosition: string) =>
+    updateSetupPosition({
+      variables: {
+        setupPosition,
+      },
+    });
+
+  useEffect(() => {
+    setSetupPosition('finish');
+  }, []);
+
+  const handleNext = async () => {
+    await setSetupPosition('');
+    push(`/accountLists/${defaultAccountListId}/settings/integrations?setup=1`);
+  };
```

Comment:
This should go to tools and not settings correct?

Reply:
Good catch, thank you!